ci-operator/step-registry/ipi/conf/gcp/private-dns-zone/ipi-conf-gcp-private-dns-zone-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-patch.yaml"
+
+CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
+
+cat > "${PATCH}" << EOF
+platform:
+  gcp:
+    privateDNSZone: 
+      id: ${CLUSTER_NAME}-private-zone
+      project: ${CLUSTER_PVTZ_PROJECT}
```

Comment:
The value of `project` is already determinated when calling `provision-private-dns-zone-in-host-project` or `provision-private-dns-zone-in-service-project`,  so I think we can save the value of project in SHARE_DIR in the steps, get its value from there.